src/OrchardCore.Modules/OrchardCore.Roles/Recipes/RolesStep.cs

```diff
@@ -35,7 +35,7 @@ public async Task ExecuteAsync(RecipeExecutionContext context)
                 if (string.IsNullOrWhiteSpace(importedRole.Name))
                     continue;
 
-                var role = (Role)await _roleManager.FindByNameAsync(importedRole.Name);
+                var role = (Role)await _roleManager.FindByNameAsync(_roleManager.NormalizeKey(importedRole.Name));
```

Comment:
I think the RoleManager implementation is supposed to do the normalization by itself.

Reply:
https://github.com/dotnet/aspnetcore/blob/master/src/Identity/Extensions.Core/src/RoleManager.cs#L326

Reply:
```suggestion
                var role = (Role)await _roleManager.FindByNameAsync(importedRole.Name);
```

Reply:
Yes else you could fake a role by using different cases on the rolename.